doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -290,6 +292,25 @@ SELECT total - progress AS remaining
 FROM system_views.sstable_tasks;
 ....
 
+=== Recent Gossip State Transitions Virtual Table
+
+The `gossip_state_transitions` virtual table provides information about
+the most recent Gossip state transitions recorded per peer. The number
+of transitions per peer can be configured by the
+`max_gossip_state_transitions_size` property, which defaults to 10.
+This table only considers peers in the ring (it doesn't track transitions
+for evicted peers).
+
+....
+cqlsh:system_views> SELECT * FROM gossip_state_transitions;
+
+ address   | port | timestamp                       | dc   | generation | heartbeat | host_id | hostname  | internal_address_and_port | internal_ip | load | native_address_and_port | net_version | rack | release_version | removal_coordinator | rpc_address | rpc_ready | schema | severity | sstable_versions | status | status_with_port | tokens
```

Comment:
This table format has many columns, only one row. It's very difficult to map column name with corresponding value, as wrapping to multiple lines. I feel having them as rows will make viewing and also scrolling comfortable. What do you think?

Reply:
Would like to have @yifan-c to comment as we recently created few virtual tables with metrics as rows.

Reply:
The table has multiple rows, potentially thousands. I think we can query a subset of the columns for documentation purposes

Reply:
Oh yeah, missed this